src/PhpSpec/Formatter/JUnitFormatter.php

```diff
@@ -52,7 +52,7 @@ class JUnitFormatter extends BasicFormatter
         ExampleEvent::SKIPPED => 'skipped',
     );
 
-    public function __construct(PresenterInterface $presenter, IO $io, StatisticsCollector $stats)
+    public function __construct(Presenter $presenter, IO $io, StatisticsCollector $stats)
```

Comment:
@ciaranmcnulty there is already the new interface and the old one in parallel in the codebase, with the old one being deprecated and inheriting from the new one. But as all typehints are using the child interface, the new interface is actually unusable today. So typehints must be changed in 2.x to allow using the existing ``Presenter`` interface (using the deprecated one will work, as it extends the new one)